### PR TITLE
Interface: Checkboxes Relational

### PR DESCRIPTION
**Pull request by theharshin**
**Options**
- Grid: Number of items to show in a single row.
- Template: How to display a relational item when single item is loaded.
- Listing Template: How to display a relational item when shown in listing.

**Issues**
The relational items can not be deleted in random order. You can only delete the last item from the option or a list of items selected from last. It's kind of last in first out problem 🤔 
This issue is with M2M interface also. Let me know if you can reproduce this issue. I'll open a seperate ticket for it.

### CODE REVIEW

**Comment by benhaynes** [comment]
Thanks @theharshin!!! Rijk has been busy with Cloud, but we'll get this reviewed Friday.

**Comment by benhaynes** [comment]
@theharshin .. did you add a ticket  for the delete issue?

**Comment by theharshin** [comment]
@benhaynes I talked with @WellingGuzman and he can't reproduce the issue! So if you can take a clone and have a look if the issue appears. I'll create a fresh collection and check if that issue is collection specific.

File: extensions/core/interfaces/checkboxes-relational/input.vue [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      let isSet = false;
+      //If the value is set
+      //We check throug items if the selected is same.
+      if (this.value) {
+        this.value.forEach(item => {
+          //If the item is deleted
+          //Restore the value from unchecked array
+          if (item.$delete) {
+            let uncheckedItem = this.unchecked[item[this.junctionPk]];
+            console.log("UNCHECKED>", uncheckedItem);
```
Comment (rijkvanzanten) [inline extensions/core/interfaces/checkboxes-relational/input.vue]: ⚠️ Spotted one! 😁

Comment (rijkvanzanten) [reply extensions/core/interfaces/checkboxes-relational/input.vue]:
```suggestion

```

**Comment by rijkvanzanten** [comment]
I haven't been able to fully test it yet, but spotted a leftover console.log on a quick pass :) 

Also @theharshin, I think you should be able to run `npm run fix` from the `extensions` folder which should fix tiny code style things (like missing / trailing spaces) and show warnings for these kind of issues 🙂

**Comment by benhaynes** [comment]
Thanks @theharshin — were you able to replicate the issue with the standard M2M on the demo site? That's the easiest way for us to replicate the problem and resolve it.

https://directus.app

**Comment by theharshin** [comment]
@benhaynes I can reproduce the issue on the directus.app with Latest Directus build. I've sent a video on Slack to quickly demo the issue.

File: extensions/core/interfaces/checkboxes-relational/input.vue (hunk 1 above)
Comment (theharshin) [reply extensions/core/interfaces/checkboxes-relational/input.vue]: This is embarrassing 😢Let me update it.

**Comment by theharshin** [comment]
@rijkvanzanten Updated the code and did the style fix.

**Pull request merged**